core/example/CMakeLists.txt

```diff
@@ -5,4 +5,6 @@ endif()
 
 add_subdirectory(benchmark)
 add_subdirectory(tutorial)
+add_subdirectory(solvers)
```

Comment:
As this is only an example this is fine for now but ultimately I think we want a new solvers package in which the code in these examples is developed into a library capability.